book/src/raw-block.md

```diff
@@ -1,8 +1,8 @@
-# Raw Block Volume Feature
+﻿# Raw Block Volume Feature
 
 > ## *This page is still under active development.*
 
-**Status:** Alpha
+**Status:** Beta since 1.14 (Alpha since 1.11)
```

Comment:
K8s 1.14  (just so there's no confusion with sidecar versions)

Reply:
Actually let's follow this format: https://github.com/kubernetes-csi/docs/pull/121/files#diff-64e1d803bf8d66339de7965984ce84ccR5